### PR TITLE
dpl: add verbose flag to filler_placement to print filler cell usage

### PR DESCRIPTION
**Pull request by gadfort**
Adds:
- -verbose to filler_placement to print a report of which cells were inserted.

### CODE REVIEW

**Review by maliberty — CHANGES_REQUESTED** [review]
This is similar to dbSta::report_cell_usage - perhaps there is some sharing possible?

File: src/dpl/test/regression_tests.tcl [hunk 1]
```diff
@@ -23,6 +23,8 @@ record_tests {
   fillers7
   fillers8
   fillers9
+  fillers2_verbose
+  fillers9_verbose
```
[the remark this answers is not in the document]
Comment (gadfort) [reply src/dpl/test/regression_tests.tcl:27]: When are we retiring the two lists?

**Comment by gadfort** [comment]
@maliberty Not sure what sharing would make sense but we can explore that as a separate issue.

**Review by maliberty — APPROVED** [review]

File: src/dpl/test/regression_tests.tcl (hunk 1 above)
[the remark this answers is not in the document]
Comment (maliberty) [reply src/dpl/test/regression_tests.tcl:27]: When someone gets around to making the flow tests work from ctest as well.

**Pull request merged**